config/activitylog.php

```diff
@@ -42,4 +42,9 @@
      * used by the Activity model shipped with this package.
      */
     'table_name' => 'activity_log',
+
```

Comment:
Still not sure weather scope fields should be "whitelisted" in the config or to give responsibility to the end user to set correct fields.